network_check.sh

```diff
@@ -15,7 +17,7 @@ clear
 ###
 
 # Set gateway_ip to the gateway that you want to check to declare network working or not
-gateway_ip='www.google.com'
+gateway_ip='1.1.1.1'
```

Comment:
IMHO:  No need to restart things because of a DNS failure.

Reply:
Thanks for the contribution, @mpatnode!

All very good points and, to be honest...

I think we should get rid of all the requirements possible and make the code as universally working as possible.
What do you think?